documentation/book/proc-downloading.adoc

```diff
@@ -0,0 +1,13 @@
+include::common/attributes.adoc[]
+
+[id='downloading-{context}']
+= Downloading {ProductName}
+
+The release artifacts contain documentation and example YAML files for deployment on {ProductPlatformName}. The example files are used throughout this documentation and can be used to install {ProductName}.
+
+The Docker images are available on {DockerRepository}.
```

Comment:
We do not really distribute any binaries as zip files. Because the whole Strimzi project runs either on Kubernetes or OpenShift, the actually software is distributed as Docker images. When users download the release zip file of Strimzi, all the get is the documentation and example YAML files which can be used to deploy the software. But the actual software will be downloaded form Docker Hub.

However, the user doesn't really download it. The YAML files basically point OpenShift or Kubernetes to the Docker Hub. At least in the old docu, it IMHO helped to explain what it is the user is actually downloading and where to find the software. I guess it could be of course also part of some other section or something like that, but IMHO this information should be in the docu.

Reply:
Thanks for explaining!! I will fix

Nicole